### ISSUE TITLE
Problems Installing To Xcode 7.2

### ISSUE THREAD

**Issue by MikeKogan** [NMSSH/NMSSH#163]
I give up. I have followed the instructions exactly. No matter what the linker always fails saying it cannot find NMSSH. I downloaded the zip - opened it, selected the iOS project, built it, added the framework from products to my project and voila, linker error forever. Not really sure where it is looking or what is going on. I have followed your steps for build and integration EXACTLY.

**Comment by Lejdborg** [comment]
It would help if you posted your errors.

The recommended way of installing is via CocoaPods, people having issues with the manual install in the past have found that installing through CocoaPods solves the problem.

**Comment by xRayCoder** [comment]
I got the same error with Xcode 7.2.

ld: building for iOS simulator, but linking against dylib built for OSX, file '/Users/Ebo/Desktop/EboSSHToolKIt/EboSSHToolKIt/NMSSH.framework/NMSSH' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

------------------------------------------
I alse failed via CocoaPods
error: 
Analyzing dependencies
[!] Unable to satisfy the following requirements:

- `NMSSH` required by `Podfile`

Specs satisfying the `NMSSH` dependency were found, but they required a higher minimum deployment target.

I tried different Deployment target 8.0, 9.0, 9.2 , all failed.
